content/docs/start/visualization.md

```diff
@@ -0,0 +1,101 @@
+# Visualization with DVC Plots
+
+As we discussed in [experiments] section, DVC versions the experiments by
+associating parameters and metrics to them. On top of that, DVC can generate
+plots and tracks plot images you generated from the code. As in other aspects of
+the experiments, DVC is flexible and provides more than one option to fit
+visualization to your workflow:
+
+- DVC can generate HTML files that includes interactive [Vega-Lite] plots, from
+  the data series in JSON, YAML, CSV, or TSV.
```

Comment:
They can all generate HTML files, right? I would focus on being able to visualize data that you save, in contrast to images, which may be better suited if either you aren't interested in keeping the underlying data or you have more complex visualization needs.